src/lib/components/inventory/list_item_modal.ts

```diff
@@ -354,17 +382,20 @@ export class ListItemModal extends FloatElement {
                                 <a
                                     href="https://csfloat.com/item/${this.listingId}"
                                     target="_blank"
-                                    class="base-button secondary-button success-link"
+                                    class="base-button secondary-button"
                                 >
                                     View ${this.listingType === 'buy_now' ? 'Listing' : 'Auction'}
                                 </a>
                                 <a
                                     href="https://csfloat.com/stall/me"
                                     target="_blank"
-                                    class="base-button secondary-button success-link"
+                                    class="base-button secondary-button"
                                 >
                                     View Your Stall on CSFloat
                                 </a>
+                                <a href="${this.searchUrl}" target="_blank" class="base-button secondary-button">
```

Comment:
They may want a way to search these items without having to list the item first -- perhaps there's a way to put a UI element somewhere (maybe to the right of the recommended price above the input).